exercises/tournament/canonical-data.json

```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "tournament",
-  "version": "1.1.0",
+  "version": "1.3.0",
```

Comment:
Ah I see. I don't really have a problem with using 1.3.0, I just didn't understand. It's fine to leave it as is